dev/tox.ini

```diff
@@ -51,4 +51,6 @@ exclude =
     python/pyspark/worker.pyi,
     python/pyspark/java_gateway.pyi,
     dev/ansible-for-test-node/*,
+    python/pyspark/sql/connect/proto/*,
```

Comment:
Should add a TODO with a JIRA (enabling it)

Reply:
This is generated code that we cannot re-enable.